tests/conftest.py

```diff
@@ -8,6 +8,11 @@
 load_dotenv()
 
 
+@pytest.fixture(scope="session")
+def TEST_C4C_API_URL():
+    return "https://my336539.crm.ondemand.com/sap/c4c/odata/v1/c4codataapi/"
```

Comment:
Should be read from env as the endpoint is private

Reply:
Done